src/components/Banner/Banner.tsx

```diff
@@ -93,11 +93,14 @@ export const Banner = ({
         <Button
           className={styles['banner__close-btn']}
           variant="bare"
-          aria-label={closeButtonText}
-          iconName="x"
-          iconPosition="after"
           onClick={(e: any) => onDismiss(e)}
-        />
+        >
+          <Icon
+            name="x"
+            title={closeButtonText}
```

Comment:
Something I noticed while going through this that's a little odd is sometimes the `Icon`s by themselves use `title` but in the `Button` component it's using `aria-label`. (Originally the Big Medium `Button` was using visually hidden text within the `children` and we've switched to `aria-label`, but either way it's a different solution than using `title`.) I'm thinking we should switch to using `title` on the `Icon` component (and maybe inside the `Icon` it uses `aria-label` or visually hidden text or whatever) for consistency.

Reply:
makes sense to me